regression-test/data/variant_p0/array_with_predicate.out

```diff
@@ -0,0 +1,15 @@
+-- This file is automatically generated. You should know what you did if you want to edit this
+-- !sql --
+1	{"arr":["1","2","3","4"]}
```

Comment:
If predict is disabled, why can we still find the results here?

Reply:
use expr